### ISSUE TITLE
Couple issues

### ISSUE THREAD

**Issue by Mqxbrah** [daveshap/BurnoutRecoveryIndex#3]
had to manually calculate work-life balance category score cause it wasn't there

Also generate scores button doesn't do anything on mobile, had to re enter all my answers on computer

Don't think the work life balance category is contributing to the total aggregate score currently

<img width="446" alt="Screenshot 2024-10-14 at 10 19 24 PM" src="https://github.com/user-attachments/assets/33346023-f7d1-4e47-bbfb-d7725c382cce">

Good quiz otherwise

**Comment by daveshap** [comment]
We won't be addressing mobile but I'll take a look at the other issues.

**Comment by ubergarm** [comment]
Huh, it worked from my android phone with firefox klar on a test URL behind NGINX:

https://emptyduck.com/bri.html

After clicking the `Generate Scores` button, you have to manually scroll down a bit to see it.
(i removed the debugging buttons so i wouldn't accidentally click them)

**Comment by daveshap** [comment]
Nice. Did you fix the missing category?

**Comment by ubergarm** [comment]
@daveshap Thanks, yes I just fixed it in the `.html` directly with a quick search replace.

The issue is that the naming uses a split on a dash `-` character:
```
input.name.split("-")[0]
```

So there are two categories that end up mapped to `Work` e.g.

```
Work-A-0
Work-Life Balance-A-0
```

So I did a quick search/replace in the `.html` file to change `Work-Life` to `WorkLife` and it picked it up correctly and calculates it now.
```
:%s/Work-Life/WorkLife/g
```

Thanks for putting this together!
